packages/fxa-auth-server/lib/routes/recovery-key.js

```diff
@@ -4,6 +4,9 @@
 
 'use strict';
 
+import RECOVERYKEY_DOCS from '../../docs/swagger/recovery-key-docs';
```

Comment:
I think maybe `RECOVERY_KEY` is a slight improvement over `RECOVERYKEY` from a readability standpoint?